### DISCUSSION TITLE
GoPro Digital Lenses

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#852] [Q&A]
Hi all,

It might not be a relevant question to "Labs" and I hope it is ok to find an answer here.

I've been reading materials regarding GoPro Digital Lenses and I used Linear for most of the time. I saw that linear is given in the range 19-39mm.

So, why the focal length is given in a range, not a fixed value?

Secondly, this is a digital lens, when choosing linear, how much does it crop in from 5.3K? It has to be smaller than 5312x2988 right?

**Comment by dnewman-gpsw** [comment]
As you can zoom.

Yes, sort of, maybe, not realy. The pixel information density changes, Linear is stretched out at edges.  If you want 16x9 Linear, it has to read 4:3 from the sensor, so it is not always clear cut.
